dev/devicelab/lib/framework/utils.dart

```diff
@@ -19,11 +19,20 @@ import 'framework.dart';
 String cwd = Directory.current.path;
 
 /// The local engine to use for [flutter] and [evalFlutter], if any.
-String get localEngine => const String.fromEnvironment('localEngine');
+String get localEngine =>
+    (const String.fromEnvironment('localEngine', defaultValue: 'a') ==
+            const String.fromEnvironment('localEngine', defaultValue: 'b'))
+        ? const String.fromEnvironment('localEngine')
+        : null;
```

Comment:
nit: let's expand the method body into the `{ }` block syntax and add comments explaining the usage of the magic values 'a' and 'b'.

Reply:
Done.